docs/reference/commandline/attach.md

```diff
@@ -14,6 +14,7 @@ parent = "smn_cli"
 
     Attach to a running container
 
+      --detach-keys       Specify the escape key sequence used to detach a container
```

Comment:
@vdemeester No worries. Thanks for clarifying, then...I think this:

```markdown
To override the sequence for an individual container, use the
--detach-keys="<sequence>" flag with the `docker attach` command. The format of
the `<sequence>` is either a letter [a-Z], or the `ctrl-` combined with any of
the following:


a-z (a single lowercase alpha character )
@ (ampersand)
[ (left bracket)
\\ (two backward slashes)
_ (underscore)
^ (caret)

These `a`, `ctrl-a`, `X`, or `ctrl-\\` values are all examples of valid key
sequences. To configure a different configuration default key sequence for all
containers, see [**Configuration file** section](cli.md#configuration-files).
```

You'll need to update the minor  sections also: 

```markdown
Override the key sequence for detaching a container. Format is a single character [a-Z] or ctrl-<value> where <value> is one of: a-z, @, ^, [,, or _
```